examples/cell_tracking.py

```diff
@@ -0,0 +1,176 @@
+"""
+Cell tracking example
+=====================
+
+This example demonstrates how the track layer is used
+for visualizing cell tracking data, by displaying 2D + time dataset 
+of cells with colored properties for track-id.
+
+Thanks to Dr. Alessia Ruggieri and Philipp Klein, Centre for Integrative Infectious
+Disease Research (CIID), University Hospital Heidelberg, Germany for the data.
+You can find the data on: https://doi.org/10.5281/zenodo.15597019
+
+"""
```

Comment:
```suggestion
.. tags:: visualization-advanced, layers
"""
```
Derp sorry I missed this before! You need to add some tags to the example so they show up properly in the gallery. This is making docs fail right now.
These are my suggestions, you can see existing tags here:
https://napari.org/stable/_tags/tagsindex.html

You can also add your own!

Reply:
No worries man, this is the first time I am hearing of the tags in docs and thanks for sharing. I wouldn't have guessed they were the culprit here.